test/blocks/merch-card/merch-card.test.js

```diff
@@ -11,126 +11,119 @@ setConfig(conf);
 
 loadStyle('/libs/blocks/merch-card/merch-card.css');
 
+/**
+ * runs list of assertions on the card to make card test smaller and more readable
+ * @param {*} card already initiated card
+ * @param {*} assertions 1. elements is an array of object that can contains selector
+ * (if not assertion is run against the card), attribute (object with name and value
+ * to be validated against selected element) and textContent (to be validated against
+ * selected element's text content)
+ * buttons is list of expected buttons in that order
+ */
+const expectToValidateHTMLAssertions = (card, assertions = {}) => {
```

Comment:
I'm not fond of these methods because, when a test fails, the initial output always looks the same. You have to check the stack trace to identify the specific test that failed.

Reply:
we could have a utility method as follows:

```javascript
const {
  headingM,
  bodyXS,
  promoText,
  variant,
  badgeBackground,
  badgeColor,
  badgeText,
  checkboxLabel,
} = merchCard;
``` 

Assuming that most of these getters already exist on the merch-card, we could add missing ones, and use regular `expect(variable).to.exist` notation which is easier to understand for me.

Reply:
i am not a huge fan of it too, may be i can add label to expect statements, but i really hate that repetitive code in tests we have atm even more

Reply:
with recent change i added, you'd have this (for attributes and text content, but we can enhance it for others as well). I like your proposal, but we could do that in a second time :) 

``` Plans Card > Supports EDU Plans card with stock
      AssertionError: attribute badge-background-color: expected '#EDCC2D' to equal '#EDFC2D'
      + expected - actual
      
      -#EDCC2D
      +#EDFC2D
      
```